backvendor/git_test.go

```diff
@@ -205,6 +205,7 @@ func TestGitFileHashesFromRef(t *testing.T) {
 		"100644 blob e69de29bb2d1d6434b8b29ae775ad8c2e48c5391\tignored.go",
 		"100644 blob e69de29bb2d1d6434b8b29ae775ad8c2e48c5391\tvendor/github.com/eggs/ham/ham.go",
 		"100644 blob e69de29bb2d1d6434b8b29ae775ad8c2e48c5391\tvendor/github.com/foo/bar/bar.go",
+		"100644 blob e69de29bb2d1d6434b8b29ae775ad8c2e48c5391\tvendor/github.com/foo/bar/bar baz.go",
```

Comment:
When does  ` baz.go` occur?

Reply:
Pushed a commit that includes a comment which hopefully makes it clearer:
```
// Test we can parse filenames that include spaces
```